src/ban.cpp

```diff
@@ -51,20 +49,18 @@ void BanManager::load()
 	JMutexAutoLock lock(m_mutex);
 	infostream<<"BanManager: loading from "<<m_banfilepath<<std::endl;
 	std::ifstream is(m_banfilepath.c_str(), std::ios::binary);
-	if(is.good() == false)
-	{
+	if (!is) {
 		infostream<<"BanManager: failed loading from "<<m_banfilepath<<std::endl;
 		throw SerializationError("BanManager::load(): Couldn't open file");
 	}
 	
-	while(!is.eof() && is.good())
-	{
+	while (is) {
```

Comment:
Just because some standard classes have operator overloads doesn't mean you should use them!  Isn't this less expressive than the former condition?